main.tf

```diff
@@ -169,23 +171,6 @@ module "vpc" {
 #######################
 # Security Groups
 #######################
-
-resource "aws_default_security_group" "default" {
-  count  = var.manage_default_security_group ? 1 : 0
-  vpc_id = module.vpc.vpc_id
-  tags   = merge(var.tags, local.tags, var.folder)
-
-  dynamic "ingress" {
-    for_each = var.default_security_group_rules
-    content {
-      from_port   = ingress.key
-      to_port     = ingress.key
-      cidr_blocks = ingress.value
-      protocol    = "tcp"
-    }
-  }
-}
-
```

Comment:
when `manage_default_security_group` is true, the VPC module already adopts the resource, but this resource conflicts with the adoption, this is the reason why the tags keeps showing up as changes.

Instead, we should use `default_security_group_ingress` and `default_security_group_egress` to customise the ACL rules where needed